v.alkis.buildings.import.py

```diff
@@ -538,7 +538,38 @@ def main():
         alkis_source = os.path.join(dldir, filename)
         if not os.path.isfile(alkis_source):
             grass.message(_("Downloading ALKIS building data..."))
-            response = requests.get(URL)
+            if federal_state == "Hessen":
+                # insert current date into download URL
+                # try dates of yesterday and tomorrow if its not working
+                today = datetime.now().strftime("%Y%m%d")
+                yesterday = (datetime.now() - timedelta(days=1)).strftime("%Y%m%d")
+                tomorrow = (datetime.now() + timedelta(days=1)).strftime("%Y%m%d")
+                dates = [today, yesterday, tomorrow]
+                URL = URL.replace("DATE", today)
+                trydownload = True
+                count = 0
+                count_date = 0
+                while trydownload:
+                    try:
+                        count += 1
+                        response = requests.get(URL)
+                        trydownload = False
+                    except:
+                        # retry download 10 times
+                        grass.message( ("retry download"))
```

Comment:
```suggestion
                        grass.message(_("retry download"))
```

Das `_(...)` Macro dient der Erkennung für Übersetzung (gettext).
Aber es müsste vermutlich, wie auch unten, als f-string geschrieben werden.